mne/viz/_3d.py

```diff
@@ -2600,6 +2600,25 @@ def plot_sparse_source_estimates(src, stcs, colors=None, linewidth=2,
     # Update the backend
     from .backends.renderer import _get_renderer
 
+    linestyles = [
```

Comment:
@agramfort wdyt?

Reply:
Put dashed before dotted to keep old behavior and we can merge